FAQ.md

```diff
@@ -0,0 +1,98 @@
+#Frequently Asked Questions
```

Comment:
Don't you need to add spaces between headings' `#` and text for GitHub to render the heading properly?